Utils/old_content_branch.py

```diff
@@ -0,0 +1,169 @@
+                click.secho("{} has a structure issue of file type{}. Error was: {}".format(file_path, type_of_file, str(e)), fg="red")
+                return {}
+    if type(data_dictionary) is dict:
+        return data_dictionary
+    return {}
+
+
+def get_yaml(file_path):
+    return get_file(yaml.safe_load, file_path, ('yml', 'yaml'))
+
+
+def get_json(file_path):
+    return get_file(json.load, file_path, 'json')
+
+
+def handle_yml_file(file_path, new_to_version, should_rewrite=True):
+    yml_content = get_yaml(file_path)
+    if parse_version(yml_content.get('toversion', '99.99.99')) < parse_version(new_to_version) or \
+            parse_version(yml_content.get('fromversion', '0.0.0')) > parse_version(new_to_version):
+        return False
+
+    yml_content['toversion'] = new_to_version
+    if should_rewrite:
+        with open(file_path, 'w') as f:
+            yaml.dump(yml_content, f)
+            print(f" - Updating {file_path}")
+
+    return True
+
+
+def handle_json_file(file_path, new_to_version, should_rewrite=True):
+    json_content = get_json(file_path)
+    if parse_version(json_content.get('toVersion', '99.99.99')) < parse_version(new_to_version) or \
+            parse_version(json_content.get('fromVersion', '0.0.0')) > parse_version(new_to_version):
+        return False
+
+    json_content['toVersion'] = new_to_version
+
+    if should_rewrite:
+        with open(file_path, 'w') as f:
+            json.dump(json_content, f, indent=4)
+            print(f" - Updating {file_path}")
+
+    return True
+
+
+def delete_playbook(file_path):
+    os.remove(file_path)
+    print(f" - Deleting {file_path}")
+
+    changelog_file = os.path.join(os.path.splitext(file_path)[0] + '_CHANGELOG.md')
+    readme_file = os.path.join(os.path.splitext(file_path)[0] + '_README.md')
+    if os.path.isfile(changelog_file):
+        os.remove(changelog_file)
+
+    if os.path.isfile(readme_file):
+        os.remove(readme_file)
+
+
+def delete_script_or_integration(path):
+    if os.path.isfile(path):
+        os.remove(path)
+        changelog_file = os.path.splitext(path)[0] + '_CHANGELOG.md'
+        if os.path.isfile(changelog_file):
+            os.remove(changelog_file)
+
+    else:
+        os.system(f"rm -rf {path}")
+    print(f" - Deleting {path}")
+
+
+def delete_json(file_path):
+    os.remove(file_path)
+    print(f" - Deleting {file_path}")
+
+    changelog_file = os.path.join(os.path.splitext(file_path)[0] + '_CHANGELOG.md')
+    if os.path.isfile(changelog_file):
+        os.remove(changelog_file)
+
+
+parser = argparse.ArgumentParser("Alter the branch to assign a new toVersion to all relevant files.")
+parser.add_argument('-v', '--new-to-version', help='The new to version to assign.', required=True)
+
+
+def main():
+    new_to_version = parser.parse_args().new_to_version
+    if new_to_version.count('.') == 1:
+        new_to_version = new_to_version + ".9"
+
+    click.secho("Starting Branch Editing")
```

Comment:
let's sort this like you did in validate, pack level, entity dir level, entity handling

Reply:
done